src/components/Data/Documents/ListActions.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="ListActions row actions">
     <div class="col s8">
-      <button
+      <button v-if="viewType !== 'map'"
```

Comment:
In order to keep this component more agnostic and improve encapsulation, it's better to use boolean props that explicitly tell which buttons to display (e.g. `showToggleBtn`, `showBulkDeleteBtn`, etc...), then leave to the parent the logic that sets them to `true` or `false`.